test/auth_test.exs

```diff
@@ -7,63 +7,97 @@ defmodule AntikytheraAws.Auth.InstanceProfileCredentialsTest do
   alias AntikytheraCore.Ets.SystemCache
   alias AntikytheraAws.Auth.Credentials, as: Creds
 
-  @valid_credentials      %Creds{access_key_id: "aws_access_key_id", secret_access_key: "aws_secret_access_key",
-                                 security_token: "token", source_type: :instance}
-  @role_endpoint          "http://169.254.169.254/latest/meta-data/iam/security-credentials/"
-  @test_role              "test_role"
-  @creds_endpoint         @role_endpoint <> @test_role
-  @creds_endpoint_success {:ok, %Res{body: ~S({"Code"  : "Success", "LastUpdated" : "2016-01-01T00:00:00Z",
+  @valid_credentials %Creds{
+    access_key_id: "aws_access_key_id",
+    secret_access_key: "aws_secret_access_key",
+    security_token: "token",
+    source_type: :instance
+  }
+  @role_endpoint "http://169.254.169.254/latest/meta-data/iam/security-credentials/"
+  @test_role "test_role"
+  @creds_endpoint @role_endpoint <> @test_role
+  @creds_endpoint_success {:ok,
+                           %Res{
+                             body:
+                               ~S({"Code"  : "Success", "LastUpdated" : "2016-01-01T00:00:00Z",
                                                "Token" : "token",   "Expiration"  : "2099-12-31T23:59:59Z",
                                                "Type"  : "AWS-HMAC","AccessKeyId" : "aws_access_key_id",
                                                "SecretAccessKey" : "aws_secret_access_key"}),
```

Comment:
Can we align here?

```suggestion
                               ~S({"Code"  : "Success", "LastUpdated" : "2016-01-01T00:00:00Z",
                                   "Token" : "token",   "Expiration"  : "2099-12-31T23:59:59Z",
                                   "Type"  : "AWS-HMAC","AccessKeyId" : "aws_access_key_id",
                                   "SecretAccessKey" : "aws_secret_access_key"}),
```

Reply:
I could not find a good way to align each line...
A newline is needed after the `~S(` to align each line, but in this case, the `body` will contain an extra newline.
Adding a newline elsewhere will either cause a syntax error or cause each line to not be aligned.

Reply:
Sorry, including L22 might be misleading.
My suggestion is to delete preceding whitespaces of L23, 24,25, and to leave L22 as is.
(this change aligns starts of `"Code"`, `"Token"`, `"Type"`, `"SecretAccessKey"`)
In my local environment, this seems to be possible.

Reply:
Thank you for your explanation.
I fixed in 5651de2
Is this what you intended?

Reply:
Yes. Thanks!